sources/osg/State.js

```diff
@@ -55,6 +55,9 @@ define( [
         // texture manager is referenced here because it's associated with gl object
         // of the gl context intialized with State
         this._textureManager = new TextureManager();
+
+
+        this._programCommonUniformsCache = {};
```

Comment:
@jtorresfabra @cedricpinson  It looks like the es6shim vendors might override the native Map.
But indeed, we should use native Map when possible.